### PR TITLE
[ROCm] gpu command buffer for ROCm

### PR DESCRIPTION
**Pull request by i-chaochen**
ROCm enable command buffer https://github.com/openxla/xla/commit/ec74b3b29d549ba97f499751651684800ab55aa6

@ezhulenev @akuegel Thanks in advance

### CODE REVIEW

**Review by akuegel — APPROVED** [review]

File: xla/stream_executor/rocm/BUILD [hunk 1]
```diff
@@ -6,6 +6,10 @@ load(
     "//xla/stream_executor:build_defs.bzl",
     "stream_executor_friends",
 )
+load(
+    "//xla:xla.bzl",
+    "xla_cc_test",
```
Comment (ddunl) [inline xla/stream_executor/rocm/BUILD:11]: Internally it complains this is unused

**Review by ddunl — APPROVED** [review]

**Comment by ddunl** [comment]
Unfortunately I think I also have to ask you to rebase to help dodge an infra issue (our problem that should be fixed soon, sorry)

**Comment by i-chaochen** [comment]
Done

**Review by ddunl — APPROVED** [review]

**Comment by ddunl** [comment]
Should be merged relatively soon, sorry for the extra problems today!!

**Pull request closed without merging**